pytorch_lightning/tuner/tuning.py

```diff
@@ -33,44 +31,30 @@ def on_trainer_init(self, auto_lr_find: Union[str, bool], auto_scale_batch_size:
         self.trainer.auto_lr_find = auto_lr_find
         self.trainer.auto_scale_batch_size = auto_scale_batch_size
 
-    def setup_trainer(
+    def _tune(
         self,
-        model: LightningModule,
-        train_dataloader: Optional[DataLoader] = None,
-        val_dataloaders: Optional[Union[DataLoader, List[DataLoader]]] = None,
-        datamodule: LightningDataModule = None,
-    ):
-        self.trainer.model_connector.copy_trainer_model_properties(model)
-        # setup data, etc...
-        self.trainer.train_loop.setup_fit(model, train_dataloader, val_dataloaders, datamodule)
-        # hook
-        self.trainer.data_connector.prepare_data(model)
+        model: 'pl.LightningModule',
+        scale_batch_size_kwargs: Optional[Dict[str, Any]] = None,
+        lr_find_kwargs: Optional[Dict[str, Any]] = None,
+    ) -> Dict[str, Optional[Union[int, _LRFinder]]]:
+        scale_batch_size_kwargs = scale_batch_size_kwargs or {}
+        lr_find_kwargs = lr_find_kwargs or {}
```

Comment:
this function is called as:
```python
result = self.tuner._tune(
            model, scale_batch_size_kwargs=scale_batch_size_kwargs or {}, lr_find_kwargs=lr_find_kwargs or {}
        )
```
the `kwargs or {}` therefore seems redundant.

Reply:
I'll remove it from the caller instead